theme-cache/web/theme.compiled.css

```diff
@@ -13642,7 +13642,6 @@ body {
   position: absolute;
   top: 30px;
   right: 30px;
```

Comment:
Should the theme-cache changes be committed?

Reply:
No, not required, I will remove this folder as well